ts/components/conversation/media-gallery/MediaGridItem.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+
+import { Message } from './propTypes/Message';
+
+interface Props {
+  message: Message;
+  onClick?: () => void;
+}
+
+const size = {
+  width: 94,
```

Comment:
I guess these numbers are all right there and nobody else needs to know about them, so it's cool. We'll have a lot more of this with CSS-in-JS. So I guess it's fair to refactor into variables on a needs-to-know (or repeated-too-often) basis?

Reply:
Yep, I think that’s a good approach.